tests/test_all_fields.py

```diff
@@ -406,10 +408,14 @@ def all_fields_test(self, streams_to_test):
                         #      we will log the inequality and assert that the datatypes at least match.
 
                         for field in set(actual_record.keys()).difference(field_adjustment_set):  # skip known bugs
+
                             with self.subTest(field=field):
                                 base_err_msg = f"Stream[{stream}] Record[{pks_tuple}] Field[{field}]"
 
                                 expected_field_value = expected_record.get(field, "EXPECTED IS MISSING FIELD")
+                                if field == "price" and expected_field_value != "EXPECTED IS MISSING FIELD" and expected_field_value.get('created'):
```

Comment:
@prijendev Why do we check `field == "price"` in this case?

Reply:
Here, it compares the field value of records retrieved through API(expected records) and records written in the target(during sync mode - actual records). `price` field is an object and contains a nested field `created`. API returns the value of `created` in timestamp while tap transforms it into the date-time format("%Y-%m-%dT%H:%M:%S.000000Z"). That's, why here we are converting the value of the `price. created` field returned by API into the date-time format.

Reply:
Add a code comment that this check is specific for "invoice_line_items" stream for better understanding

Reply:
Removed these changes and moved the `price` field in `records_data_type_conversions` to convert timestamp value to date-time format. Added comment in the code. [Reference](https://github.com/singer-io/tap-stripe/blob/master/tests/base.py#L380)